tests/integ/conftest.py

```diff
@@ -61,6 +61,7 @@ def connection(db_parameters):
         database=ret["database"],
         account=ret["account"],
         protocol=ret["protocol"],
+        role=ret.get("role"),
```

Comment:
It'd better to have this change as a separate PR.

Reply:
I'll move it to a separate PR. Just a note that our test need this change to run properly when we are using our preprod12 account

Reply:
https://github.com/snowflakedb/snowpark-python/pull/706